rasa/core/channels/slack.py

```diff
@@ -387,6 +416,35 @@ def get_metadata(self, request: Request) -> Dict[Text, Any]:
 
         return {}
 
+    def is_request_from_slack_authentic(self, request: Request) -> bool:
+        """Validate a request from Slack for its authenticity.
+
+        Checks if the signature matches the one we expect from Slack. Ensures
+        we don't process request from a third-party disguising as slack.
+
+        Args:
+            request: incoming request to be checked
+
+        Returns:
+            `True` if the request came from Slack.
+        """
+        if self.slack_signing_secret is None:
+            # this is done for backwards compatibility, but will be removed in
```

Comment:
optional: Should we add a todo we don't forget? And do you create the removal PR against `master`? Alternatively add an issue to the milestone please.
```suggestion
            # TODO: Remove when releasing 2.1.0
            # this is done for backwards compatibility, but will be removed in
```

Reply:
yes I'll create another PR against master :+1: